packages/bvaughn-architecture-demo/components/sources/Source.tsx

```diff
@@ -98,6 +104,9 @@ export default function Source({
                 data-test-id={`SourceLine${lineNumber}`}
               >
                 <div className={styles.LineNumber}>{lineNumber}</div>
+                <div
+                  className={`${styles[`LineHitCount-${hitCountBadge}`]} ${styles.LineHitCount}`}
+                ></div>
```

Comment:
Something about this is breaking on hover styles:
![Screen Shot 2022-07-05 at 8 14 50 AM](https://user-images.githubusercontent.com/29597/177324980-cec3b9ce-0eba-439c-bff3-339097a699fe.png)

Not sure what yet.